uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1225,9 +1587,17 @@ sub validateCandidate {
    return $CandMismatchError;
 }
 
-################################################################
-#################### computeSNR ################################
-################################################################
+
+=pod
+
+=head3 computeSNR($tarchiveID, $tarchive_srcloc, $profile)
+
+Computes the SNR on the modalities specified in the Config table for a given
```

Comment:
specified in the  `getSNRModalities()` routine of the `$profile` file

Reply:
oups... I thought it was in the Config table but confused it with Imaging Browser Tabulated Scan Types.

I just looked at the function and wonder if we could not have that in the Config table instead of a function in the prod file. I think I'll create a ticket on redmine to see if there are any other functions that could be moved to the Config table. I think most of them could be removed from the prod file and put into the code base with the regex they use grepped from the Config module.

Reply:
I think it would be great! but again, same comment on the regex I put on your other Loris pull request; we should avoid regex if we can and use the Config way of adding using dropdowns (like we do for double data entry instruments). In this case, it should be done this way as the modalities you want to pick from are already in the mri_scan_type table. 
just my opinion!

Reply:
even better :)